catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/source/WfsSource.java

```diff
@@ -341,7 +341,7 @@ private boolean hasUsernameChanged(String username) {
     }
 
     private boolean hasDisableCnCheck(Boolean disableCnCheck) {
-        return this.disableCnCheck != disableCnCheck;
+        return this.disableCnCheck.booleanValue() != disableCnCheck.booleanValue();
```

Comment:
Tried suppressing the warning since we may have unnecessary logic needed to resolve this.